src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/commands.py

```diff
@@ -68,8 +68,9 @@ def transform_list_location_output(result):
 cli_command(__name__, 'appservice web deployment slot create', 'azure.cli.command_modules.appservice.custom#create_webapp_slot')
 
 cli_command(__name__, 'appservice web deployment user set', 'azure.cli.command_modules.appservice.custom#set_deployment_user')
-cli_command(__name__, 'appservice web deployment list-site-credentials',
-            'azure.mgmt.web.operations.web_apps_operations#WebAppsOperations.list_publishing_credentials', factory)
+cli_command(__name__, 'appservice web deployment list-publishing-profiles',
+            'azure.cli.command_modules.appservice.custom#list_publish_profiles')
```

Comment:
It seems like this new command returns more things in a better format. Do we need to change the name? We should have a discussion on how to deprecate commands within the CLI.  cc/ @johanste